manifests/r/Ripose/Memento/1.2.0/Ripose.Memento.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Ripose.Memento
+PackageVersion: 1.2.0
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+FileExtensions:
+- mkv
+- mp4
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2010.x86

```

```suggestion
  - PackageIdentifier: Microsoft.VCRedist.2010.x64
```
From the error, I think the dependency might need to be 64-bit.